src/VolleyManagement.Backend/VolleyManagement.Services/PlayerService.cs

```diff
@@ -75,9 +75,9 @@ public void Create(Player playerToCreate)
             {
                 throw new ArgumentNullException("playerToCreate");
             }
-
-            _playerRepository.Add(playerToCreate);
-            _playerRepository.UnitOfWork.Commit();
+             
+            _playerRepository.Add(playerToCreate.FirstName, playerToCreate.LastName, 
```

Comment:
We should not accept Domain instance here. It might give a false impression that it's already instantiated